test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -93,7 +91,11 @@ trait CollectionTest[F <: CollectionFactory] extends JasmineTest {
     }
 
     it("should store custom objects") {
-      case class TestObj(num: Int)
+      case class TestObj(num: Int) extends jl.Comparable[TestObj] {
+        def compareTo(o: TestObj): Int =
+          if (o == null) -1
+          else num.compareTo(o.num)
```

Comment:
I thought I asked that earlier, but I can't find the comment anymore.
Comparing `o == null` seems very suspicious to me, for an implementation of `compareTo`. `compareTo` should not be called with `o == null` in the first place, so testing against `null` is superfluous (it might even hide a bug elsewhere).